### PR TITLE
Add Suade job post for Senior Developer

### PR DESCRIPTION
**Pull request by jesper**

### CODE REVIEW

File: jobs/suade_senior_developer.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+title: Senior Developer
+company: Suade Labs
+location: London, England
+url: https://suade.org/jobs
+contract: permanent
+contact:
+name: Jesper Thomschutz
```
Comment (salimfadhley) [inline jobs/suade_senior_developer.html]: This looks wrong - name and email should be indented. You need to follow the template exactly!

File: jobs/suade_senior_developer.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+email: jobs@suade.org
+created: !!timestamp '2017-01-13'
+tags:
+- london
+- uk
+- flask
+- postgres
+- docker
+- senior
```
Comment (salimfadhley) [inline jobs/suade_senior_developer.html]: Consider adding the main technologies you use as tags.

File: jobs/suade_senior_developer.html [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+Experience working with very large data-sets is a big plus.
+
+Knowing how to spot performance bottle-necks as well as analyzing memory usage is a great skill that we'd love for our candidate to have!
+
+
+
+# About Suade
+
+We are an early-stage FinTech startup based in London. We have had the pleasure and good fortune of being a part of the Techstars and Microsoft Ventures programs. We also have a partnership with Simmons & Simmons, who are a top-tier law firm in the Financial and Technology sectors. The company was founded on the belief that there must be a better way for financial institutions to comply with complex regulations. Our journey has led us through over 500 product iterations as a result of our conversations with experts on both sides of the rules. Our solution draws on innovative tech concepts from industries as diverse as transportation, social media, e-commerce and graphic design. We are uniquely positioned with a wealth of banking and technical talent to solve this real-world problem and deliver it to the industry.
```
Comment (salimfadhley) [inline jobs/suade_senior_developer.html]: Can you escape the ampersand - or replace it with an "and"

File: jobs/suade_senior_developer.html [hunk 4]
```diff
@@ -0,0 +1,62 @@
+---
+title: Senior Developer
+company: Suade Labs
+location: London, England
```
Comment (salimfadhley) [inline jobs/suade_senior_developer.html]: You can add a more specific location, for example Camden, London, England.

**Comment by jesper** [comment]
Thanks @salimfadhley for the review! Update pushed which should fix the issues!

**Comment by salimfadhley** [comment]
I've pushed your changes - please feel free to resubmit if you are not happy.

**Comment by jesper** [comment]
Cheers, much appreciated!

**Pull request merged**